### PR TITLE
Continue ci_script implementation replacing shell and command built-i…

### PR DESCRIPTION
**Pull request by adrianfusco**
Continue ci_script implementation replacing shell and command built-in modules

As a pull request owner and reviewers, I checked that:
- [X] Appropriate documentation exists and/or is up-to-date:

### CODE REVIEW

**Comment by dasm** [comment]
Hi @adrianfusco Why are we doing that? What are the pros/cons of `ci_script` for single line command?

**Comment by adrianfusco** [comment]
> Hi @adrianfusco Why are we doing that? What are the pros/cons of `ci_script` for single line command?

Hi @dasm. There's a task (OSP-26801) for replacing all the `command` and `shell` ones with the `ci_script.py` action plugin since it provides logs.

> Hi @adrianfusco Why are we doing that? What are the pros/cons of `ci_script` for single line command?

**Comment by dasm** [comment]
Thanks @adrianfusco for additional information.

**Review by dasm — APPROVED** [review]
/lgtm

**Comment by adrianfusco** [comment]
recheck

File: ci_framework/roles/ci_setup/tasks/packages.yml [hunk 1]
```diff
@@ -24,8 +24,9 @@
       register: oc_version
       environment:
         PATH: "{{ cifmw_path }}"
-      ansible.builtin.command:
-        cmd: "oc version -o yaml"
+      ci_script:
```
Comment (cjeanner) [inline ci_framework/roles/ci_setup/tasks/packages.yml:27]: not sure `ci_script` adds any value here. Getting the whole reproducer + log file just for this `oc version` isn't worth it imho

File: ci_framework/roles/repo_setup/tasks/rhos_release.yml [hunk 2]
```diff
@@ -7,7 +7,9 @@
 
 - name: Get rhos-release tool version
   become: true
-  ansible.builtin.command: rhos-release --version
+  ci_script:
```
Comment (cjeanner) [inline ci_framework/roles/repo_setup/tasks/rhos_release.yml:10]: same, not sure about the value to switch to ci_script

File: ci_framework/roles/ci_setup/molecule/default/converge.yml [hunk 3]
```diff
@@ -61,8 +62,9 @@
           register: oc_ver
           environment:
             PATH: "{{ cifmw_path }}"
-          ansible.builtin.command:
-            cmd: "oc version -o yaml"
+          ci_script:
```
[the remark this answers is not in the document]
Comment (adrianfusco) [reply ci_framework/roles/ci_setup/molecule/default/converge.yml]: I've reverted it. Thx.

File: ci_framework/roles/ci_setup/tasks/packages.yml (hunk 1 above)
Comment (adrianfusco) [reply ci_framework/roles/ci_setup/tasks/packages.yml]: Got it. I won't change for simple commands.

File: ci_framework/roles/repo_setup/tasks/rhos_release.yml (hunk 2 above)
Comment (adrianfusco) [reply ci_framework/roles/repo_setup/tasks/rhos_release.yml]: Ack. Reverted.

**Comment by adrianfusco** [comment]
recheck

**Review by marios — APPROVED** [review]
looks reasonable

**Comment by adrianfusco** [comment]
recheck

**Review by cjeanner — APPROVED** [review]

**Pull request merged**